packages/xo-web/src/common/render-xo-item.js

```diff
@@ -141,30 +140,36 @@ export const PoolItem = [
 
 PoolItem.propTypes = XO_ITEM_PROP_TYPES
 
-// ===================================================================
-
 // Host, Network, VM-template.
-const PoolObjectItem = propTypes({
-  object: propTypes.object.isRequired,
-})(
+export const PoolObjectItem = [
```

Comment:
Why don't we create 3 components instead? I think we should stop trying to mutualize everything at this level and create small specific components for each type of objects meant to be displayed in the UI. There aren't thousands of them anyway and it would be easier if we want to change something on one of them only. And it will avoid stuff like `OBJECT_TYPE_TO_ICON`.

Reply:
See #3303, #3302